<launch>
  <rosparam file="$(find kinematic_calibration)/config/params_general.yaml" command="load"/>
  <node pkg="kinematic_calibration" type="optimizationNode" name="optimizationNode" output="screen" >
  <rosparam file="$(find kinematic_calibration)/config/nao_calibration.yaml" />
</node>
  <node pkg="kinematic_calibration" type="updateNode" name="updateNode" />
</launch>
